src/main/target/MATEKF722/target.c

```diff
@@ -30,7 +30,7 @@
 const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
     DEF_TIM(TIM9, CH2, PA3, TIM_USE_PPM, 0, 0),      // PPM
 
-    DEF_TIM(TIM8, CH1, PC6, TIM_USE_MOTOR, 0, 0),   // S1   DMA2_ST2 (XXX was DMA1_ST4)
+    DEF_TIM(TIM8, CH1, PC6, TIM_USE_MOTOR, 0, 1),   // S1   DMA2_ST2 (XXX was DMA1_ST4)
```

Comment:
Can we have this in a separate pull request please, so we can track issue reports (if any) against it separately.

Reply:
You bet!